### PR TITLE
Use engine names in the PR template

### PR DESCRIPTION
**Pull request by annevk**
This seems more accurate.

### CODE REVIEW

**Comment by annevk** [comment]
Any other changes we should make before pushing out a bunch of PRs?

**Comment by annevk** [comment]
@jgraham thoughts on this?

File: PULL_REQUEST_TEMPLATE.md.template [hunk 1]
```diff
@@ -8,8 +8,8 @@ Thank you for contributing to the @@h1@@ Standard! Please describe the change yo
 - [ ] [Tests](https://github.com/web-platform-tests/wpt) are written and can be reviewed and commented upon at:
    * …
 - [ ] [Implementation bugs](https://github.com/whatwg/meta/blob/main/MAINTAINERS.md#handling-pull-requests) are filed:
-   * Chrome: …
-   * Firefox: …
-   * Safari: …@@extra_implementers@@
+   * Chromium: …
```
Comment (andreubotella) [inline PULL_REQUEST_TEMPLATE.md.template:11]:
```suggestion
   * Blink: …
```

Comment (annevk) [reply PULL_REQUEST_TEMPLATE.md.template:11]: Yeah I wasn't sure about that one. Non-Chrome-but-Chrome-like browsers typically don't say they are based on Blink. And most of the infrastructure apart from blink-dev also talks about Chromium. That's why I went with that.

**Comment by jgraham** [comment]
Seems reasonable to me.

**Review by domenic — APPROVED** [review]
LGTM, preferring Chromium to Blink

**Pull request merged**